.github/workflows/ci.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       - name: Install doc dependencies
         run: |
           sudo apt update --yes && sudo apt install --yes git build-essential pandoc graphviz
-          pip install -U pip towncrier
+          pip install -U pip towncrier myst-nb
```

Comment:
should we not include those now in `setup.cfg` under `options.extras_require.docs`?